integration-tests/bats/sql-spatial-types.bats

```diff
@@ -20,4 +20,31 @@ teardown() {
     DOLT_ENABLE_SPATIAL_TYPES=true run dolt sql -q "create table point_tbl (p point)"
     [ "$status" -eq 0 ]
     [[ "$output" = "" ]] || false
+}
+
+@test "sql-spatial-types: create point table and insert point" {
+    DOLT_ENABLE_SPATIAL_TYPES=true run dolt sql -q "create table point_tbl (p point)"
+    [ "$status" -eq 0 ]
+    [[ "$output" = "" ]] || false
+    DOLT_ENABLE_SPATIAL_TYPES=true run dolt sql -q "insert into point_tbl () values (point(1,2))"
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "Query OK" ]] || false
+}
+
+@test "sql-spatial-types: create linestring table and insert linestring" {
+    DOLT_ENABLE_SPATIAL_TYPES=true run dolt sql -q "create table line_tbl (l linestring)"
+    [ "$status" -eq 0 ]
+    [[ "$output" = "" ]] || false
```

Comment:
This is a useless test. Your regexing an empty string. All strings contain an empty string :-)

Reply:
Then you need to do `[ "$output" = "" ]`. Not sure what  the regex version does in this context.

Reply:
https://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash#:~:text=Double%20Brackets%20i.e.%20%5B%5B%5D%5D,%2C%20literally.

A bit of a deep dive if you're curious. Turns out they are equivalent in this case but I prefer the single brackets fro equality because you don't need the || false for the ancient mac version of bash to work consistently.

Reply:
Ok, will change